contrib/sssd.spec.in

```diff
@@ -35,7 +35,7 @@ Requires: sssd-krb5 = %{version}-%{release}
 Requires: sssd-ipa = %{version}-%{release}
 Requires: sssd-ad = %{version}-%{release}
 Recommends: sssd-proxy = %{version}-%{release}
-Suggests: python3-sssdconfig = %{version}-%{release}
+Requires: python3-sssdconfig = %{version}-%{release}
 Suggests: sssd-dbus = %{version}-%{release}
```

Comment:
Is it really 'Suggests' and not 'Requires'?

Reply:
There is suggest in Fedora. It is missing in RHEL. So it definitely should not be required, there is no reason for it. It is perfectly fine to run sssd without sssd-dbus (ifp responder).

Reply:
Ah, I got purpose of this package wrong, sorry.